docs/content/preview/yugabyte-voyager/migrate/live-fall-forward.md

```diff
@@ -0,0 +1,573 @@
+---
+title: Steps to perform live migration of your database using YugabyteDB Voyager
+headerTitle: Live migration with fall-forward
+linkTitle: Live migration with fall-forward
+headcontent: Steps for a live migration with fall-forward using YugabyteDB Voyager
+description: Steps to ensure a successful live migration with fall-forward using YugabyteDB Voyager.
+menu:
+  preview_yugabyte-voyager:
+    identifier: live-fall-forward
+    parent: migration-types
+    weight: 103
+techPreview: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+rightNav:
+  hideH4: true
+type: docs
+---
+
+When you migrate a database using YugabyteDB Voyager, it is prudent to have a backup strategy if the new database doesn't work as expected. A fall-forward approach consists of creating a third database (Fall-forward database) that is a replica of your original source database.
+
+A fall forward approach allows you to test the system end-to-end. This workflow is especially important in heterogeneous migration scenarios, in which source and target databases are using different engines.
+
+## Fall-forward workflow
+
+![fall-forward short](/images/migrate/live-fall-forward-short.png)
+
+After starting [live migration](../live-migrate/#live-migration-workflow) of data, an additional step of setting up the fall-forward database (via [fall-forward setup](#fall-forward-setup)) needs to be performed. The setup starts replicating the snapshot data along with new changes exported from the source database to the fall-forward DB, as shown in the following illustration:
+
+![After fall-forward setup](/images/migrate/after-fall-forward-setup.png)
+
+At [cutover](#cutover), applications stop writing to the source database and start writing to the target YugabyteDB. After the cutover process is complete, YB Voyager keeps the fall-forward database synchronized with changes from the target Yugabyte DB as shown in the following illustration:
+
+![After cutover](/images/migrate/after-cutover.png)
+
+Finally, if you must switch to the fall-forward database in case the current YugabyteDB system is not working as expected, you can use [fall-forward switchover](#fall-forward-switchover-optional) option to achieve the same.
```

Comment:
```suggestion
Finally, if you need to switch to the fall-forward database (because the current YugabyteDB system is not working as expected), you can use [fall-forward switchover](#fall-forward-switchover-optional) option.
```